src/core/Akka.Remote.Tests/RemotingSpec.cs

```diff
@@ -14,16 +14,13 @@
 using Akka.Remote.Transport;
 using Akka.Routing;
 using Akka.TestKit;
-using Akka.TestKit.TestEvent;
-using Akka.Util;
```

Comment:
Formatted this file and deleted the only remaining `Resolver` usages here - tests still passed without it.